src/firewall/iptables.rs

```diff
@@ -285,6 +395,21 @@ fn chain_exists(driver: &IPTables, table: &str, chain: &str) -> Result<bool, Box
     serde::__private::Result::Ok(false)
 }
 
+fn remove_if_exists(
```

Comment:
function name could be more explicit `remove_rule_if_exists` I think would help with readability.